### PR TITLE
add default values to ldap_client

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: lib/puppet/type/netapp_ldap_client.rb [hunk 1]
```diff
@@ -69,6 +70,7 @@
 simple - Simple bind,
 sasl - Simple Authentication and Security Layer (SASL) bind"
     newvalues(:anonymous, :simple, :sasl)
+    defaultto(:simple)
```
Comment (hunner) [inline lib/puppet/type/netapp_ldap_client.rb]: The copy/pasted docs mention various defaults under different situations. Either the code should match this, or the docs should be updated.

Comment (hunner) [inline lib/puppet/type/netapp_ldap_client.rb]: Readme will need an update for this too.

File: lib/puppet/type/netapp_ldap_client.rb [hunk 2]
```diff
@@ -69,6 +70,7 @@
 simple - Simple bind,
 sasl - Simple Authentication and Security Layer (SASL) bind"
     newvalues(:anonymous, :simple, :sasl)
+    defaultto(:sasl)
```
Comment (hunner) [inline lib/puppet/type/netapp_ldap_client.rb]: Still gotta update the `desc` and readme for this.

Comment (tphoney) [inline lib/puppet/type/netapp_ldap_client.rb]: done and done

**Pull request merged**